src/analysis/rust_type.rs

```diff
@@ -232,6 +235,14 @@ impl<'env> RustTypeBuilder<'env> {
         self
     }
 
+    pub fn callback_parameters_config(
+        mut self,
+        callback_parameters_config: &[CallbackParameter],
+    ) -> Self {
+        self.callback_parameters_config = callback_parameters_config.to_owned();
```

Comment:
Why not apply the configuration to the builder here, but instead copy the configuration just to apply it at a later time?

Reply:
What do you mean?

Reply:
You're only storing the configuration in the builder here instead of changing the state of the builder (i.e. the `nullable` field)

Reply:
Oh